llpc/util/llpcCacheAccessor.h

```diff
@@ -51,6 +51,33 @@ class CacheAccessor {
     initializeUsingBuildInfo(buildInfo, cacheHash, compiler);
   }
 
+  CacheAccessor(CacheAccessor &&ca) {
```

Comment:
If you define a move constructor, one would also expect a move assignment operator.
Also, why not just:
```
CacheAccessor(CacheAccessor&&) = default;
CacheAccessor& operator=(CacheAccessor&&) = default;
```

? We use this in other places in the codebase, and so does LLVM.

Reply:
I'm not sure the default is the behaviour that I wanted.  In particular, I don't think `m_cacheResult` would get the correct value, which I want to make `ErrorUnknown`.  In any case, I do not need this anymore, so I will probably just delete it.

Reply:
So I do need the constructor because I return CacheAccessor objects.  I've created the move constructor and move assignment.  I've explicitly deleted a few constructors and assignments.  Let me know if I've missing something else I should delete.